src/etl_localization_rcnn.cpp

```diff
@@ -110,7 +110,7 @@ shared_ptr<localization::rcnn::decoded>
     }
 
     cv::Size im_size{mp->width(), mp->height()};
-    auto     crop = settings->cropbox;
+    //auto     crop = settings->cropbox;
```

Comment:
Why have you left the commented variable ?

Reply:
Well, I thought maybe its a mistake that its not used. Maybe it should be used but for some reason its not. We don't have tests enabled for it. I left it as a hint to someone who might work on it in the future. 
Should I remove it? Someone can later look at file history if its needed.

Reply:
You can keep this line but I suggest to add comment with explanation.

Reply:
I will remove it